src/core/repository.ts

```diff
@@ -19,13 +19,19 @@ export type Event<T> = {
   data?: T
 }
 
-export const createRepository = <T>(dbConn: Pool) => {
+const pool = new Pool({

```

Comment:
pool:en sätter man upp i src/server.ts "connectToDb"-funktionen. Du kan skicka in optionsobjektet med max:10, connectionString.. yada yada till den där